tests/FrontBaseClass.php

```diff
@@ -59,14 +59,26 @@ public function afterTestMethod($method)
         parent::afterTestMethod($method);
     }
 
-    protected function logIn()
+    protected function loginProvider()
+    {
+        return [
+            [TU_USER, TU_PASS],
+            ['jsmith123', TU_PASS],
+            ['john+doe@doe.com', TU_PASS],
+        ];
+    }
+
+    /**
```

Comment:
I'm not sure this is a good idea. This login method is meant to be called from subclasses several times, and we need a real login on a well known user in this case.

I guess a better solution would be to create a new method with the provider which will just call the existing one